fw/http_parser.c

```diff
@@ -8557,6 +8582,11 @@ tfw_h2_parse_req(void *req_data, unsigned char *data, size_t len,
 
 	if (likely(type != HTTP2_DATA))
 		r = tfw_hpack_decode(&ctx->hpack, data, len, req, parsed);
+	else if (unlikely(req->method == TFW_HTTP_METH_GET)
+		  || unlikely(req->method == TFW_HTTP_METH_HEAD)
+		  || unlikely(req->method == TFW_HTTP_METH_DELETE)
+		  || unlikely(req->method == TFW_HTTP_METH_TRACE))
+		r = T_DROP;
```

Comment:
We use braces for if-else with multiline conditions and there is no sense to use separate `unlikely` for each statment, should be something like:
```C
        else if (unlikely(req->method == TFW_HTTP_METH_GET
                          || req->method == TFW_HTTP_METH_HEAD
                          || req->method == TFW_HTTP_METH_DELETE
                          || req->method == TFW_HTTP_METH_TRACE))
        {
                r = T_DROP;                                                                                         
        }
```

Reply:
I see. I'll fix it.